modules/local/create_gene_heatmap.nf

```diff
@@ -22,7 +22,7 @@ process CREATE_GENE_HEATMAP {
     def gene_filter = params.heatmap_genes_to_filter ?: ""
 
     """
-    compute_gene_heatmap.R $annotated_counts $gene_filter
+    compute_gene_heatmap.R $annotated_counts $gene_filter $params.heatmap_id_column
```

Comment:
why not doing it the way you did on L22?

Reply:
Related, why not instead using ext.args for both?

Reply:
Thanks for the review and comments @maxulysse. I changed the parameter handling accordingly (and made some additional changes). Please let me know if that's not how you would use `ext.args`. :)